src/phaser/scene/VillageScene/index.js

```diff
@@ -39,36 +39,36 @@ class VillageScene extends Phaser.Scene {
 
     socket.emit('character:start', 'start');
 
-    socket.on('character:currentCharacter', characters => {
+    socket.once('character:myCharacter', myCharacterInfo => {
```

Comment:
```suggestion
    socket.once('character:createMyCharacter', myCharacterInfo => {
```

좀 뒤늦게 보인거지만, 이벤트 네이밍이 좀 더 자세했으면 좋을 것 같아요. `createMyCharacter` 와 같이 이벤트에 대한 설명을 좀 더 구체적으로 적으면 어떨까요?

Reply:
사실 이후에도 하나 작업해둔게 더 있는데 socket message를 한번 싹다 변경해야 할 것 같아서 이슈로 남겨두고 추후에 리팩터링 해도 될까요?

최대한 브랜치를 나누고 그 외의 작업을 지양하려고 하니까 이렇게 되는 것 같네요 ㅠㅠ

Reply:
아아 넵! 추후 리팩터링하는 걸로 알고 있겠습니다!